backend/src/services/secret-v2-bridge/secret-v2-bridge-dal.ts

```
@@ -142,9 +144,14 @@ export const secretV2BridgeDALFactory = ({ db, keyStore }: TSecretV2DalArg) => {
   const find = async (filter: TFindFilter<TSecretsV2>, opts: TFindOpt<TSecretsV2> = {}) => {
     const { offset, limit, sort, tx } = opts;
     try {
+      const qualifiedFilter = { ...filter };
+      if ("userId" in qualifiedFilter) {
+        delete qualifiedFilter.userId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)